crates/core/src/host/instance_env.rs

```diff
@@ -534,6 +534,9 @@ mod test {
 
     /// An `InstanceEnv` requires `ModuleSubscriptions`
     fn subscription_actor(relational_db: Arc<RelationalDB>) -> ModuleSubscriptions {
+        // Create and enter a Tokio runtime to run the `ModuleSubscriptions`' background workers in parallel.
+        let runtime = tokio::runtime::Runtime::new().unwrap();
+        let _rt = runtime.enter();
```

Comment:
Why do subscriptions need their own runtime? Also shouldn't we limit the number of threads for this runtime? The default is `num_cpu`.